app/components/primer/alpha/toggle_switch.pcss

```diff
@@ -208,6 +208,11 @@
   text-align: right;
 }
 
+.ToggleSwitch-statusIcon {
+  width: 16px;
+  height: 16px;
```

Comment:
```suggestion
  width: var(--base-size-16, 16px);
  height: var(--base-size-16, 16px);
```

Reply:
This div wraps a couple of octicons, so I think we explicitly want 16px of width and height?